pkgs/applications/misc/masterpdfeditor4/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, fetchurl, sane-backends, qtbase, qtsvg, autoPatchelfHook, lib, wrapQtAppsHook }:
+
+stdenv.mkDerivation rec {
+  pname = "masterpdfeditor4";
+  version = "4.3.89";
+
+  src = fetchurl {
+    url = "https://code-industry.net/public/master-pdf-editor-${version}_qt5.amd64.tar.gz";
+    sha256 = "0k5bzlhqglskiiq86nmy18mnh5bf2w3mr9cq3pibrwn5pisxnxxc";
+  };
+
+  nativeBuildInputs = [ autoPatchelfHook wrapQtAppsHook ];
+
+  buildInputs = [ qtbase qtsvg sane-backends stdenv.cc.cc ];
+
+  dontStrip = true;
```

Comment:
Why are we not stripping it?

Reply:
That's how it is done in the original masterpdfeditor package.

Reply:
If there is no reason to not strip like crashes we want to do this.

Reply:
Seems to be working, so I removed the `dontStrip`.